include/rangers-api/ApplicationCommon/FSM/GOCTinyFSMContext.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+namespace app_cmn::fsm
+{
+    class GOCTinyFsmContext : public hh::fnd::ReferencedObject {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.